io/src/main/scala/sbt/internal/io/Milli.scala

```diff
@@ -281,12 +290,16 @@ private object WinMilli extends MilliNative[FILETIME] {
                            null)
     if (hFile == INVALID_HANDLE_VALUE) {
       val err = GetLastError()
-      if (err == ERROR_FILE_NOT_FOUND || err == ERROR_PATH_NOT_FOUND)
+      if (err == ERROR_FILE_NOT_FOUND || err == ERROR_PATH_NOT_FOUND || err == ERROR_ACCESS_DENIED)
```

Comment:
Good catch. An alternative could be throwing `java.nio.file.AccessDeniedException` for `ERROR_ACCESS_DENIED`, but `java.io.FileNotFoundException` was the usual response before nio, so I'm ok with that. I'd just use a different string for the error message in that case, for clarity, like:
```
if (err == ERROR_FILE_NOT_FOUND || err == ERROR_PATH_NOT_FOUND)
  throw new FileNotFoundException("Not found: " + lpFileName)
else if (err == ERROR_ACCESS_DENIED)
  throw new FileNotFoundException("Access denied: " + lpFileName)
else
```